hashicorp/boundary-ui
# fix: 🐛 change delete to detach Storage policy from a scope

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12817

https://hashicorp.atlassian.net/browse/ICU-12817

## Description

change `delete` to `detach`  storage policy from a scope



## Screenshots (if appropriate):
![Untitled](https://github.com/hashicorp/boundary-ui/assets/15043878/d04a1d41-26d5-4573-bdf0-ac42150c0f66)


## How to Test
1.  go to global settings from the side right bar
2. click on attach storage policy 
3. after attaching, click the dropdown menu on the attached policy
4. it should say `detach` storage policy 





:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
